packages/app/src/ui/components/modal-dialog/get-modal-dialog-style.helper.ts

```diff
@@ -30,7 +30,7 @@ const getModalSize = (size: ModalProps["size"]) => {
     case ModalSize.Small:
       return {
         width: "38rem",
-        padding: "2.4rem 2.4rem 4rem 2.4rem",
+        padding: "2.4rem 2.4rem 2.4rem 2.4rem",
```

Comment:
The same case like above 🙄 Im not sure that we should update a core/ui styles during on this issue...